frontend/lib/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -405,11 +405,11 @@ class DateInput extends React.PureComponent<Props, State> {
                   Input: {
                     style: {
                       // Baseweb requires long-hand props, short-hand leads to weird bugs & warnings.
-                      paddingRight: ".5rem",
-                      paddingLeft: ".5rem",
-                      paddingBottom: ".5rem",
-                      paddingTop: ".5rem",
-                      lineHeight: 1.4,
+                      paddingRight: theme.spacing.sm,
+                      paddingLeft: theme.spacing.sm,
+                      paddingBottom: theme.spacing.sm,
+                      paddingTop: theme.spacing.sm,
+                      lineHeight: theme.lineHeights.inputWidget,
```

Comment:
nit: similar destructuing can just clean this up.

Reply:
Updated 👍